### PR TITLE
Upgrade to Polymer 3

### LINKED ISSUE #74: Upgrade to Polymer 3
Depends on:

* [x] cosmoz-page-router https://github.com/Neovici/cosmoz-page-router/pull/55

### PR DESCRIPTION
**Pull request by cristinecula**
Fixes #74

### CODE REVIEW

**Comment by cristinecula** [comment]
@megheaiulian @nomego

**Comment by cristinecula** [comment]
Depends on:
```
"@neovici/cosmoz-page-router": "plumelo/cosmoz-page-router#feature/NEOV-361-3.x",
```

File: cosmoz-tabbable-behavior.js [hunk 1]
```diff
@@ -0,0 +1,285 @@
...
+
+import { IronResizableBehavior } from '@polymer/iron-resizable-behavior/iron-resizable-behavior';
+import { IronMultiSelectableBehaviorImpl, IronMultiSelectableBehavior } from '@polymer/iron-selector/iron-multi-selectable';
+
+/** @polymerBehavior */
+export const TabbableBehaviorImpl = {
+	properties: {
+
+		/**
+			 * Toggles the accordion mode for the element.
```
Comment (nomego) [inline cosmoz-tabbable-behavior.js]: Weird indent on the jsdocs

File: cosmoz-tabbable-behavior.js [hunk 2]
```diff
@@ -0,0 +1,285 @@
...
+			this.set('selectedValues', [this._normalizeValue(this.selected)]);
+		} else if (multi === false && old === true && this.selectedValues[0] !== undefined) {
+			this._selectMulti([this.selectedValues[0]]);
+			this.selected = this.selectedValues[0];
+		}
+		this._updateSelected();
+	},
+
+	_normalizeValue(value) {
+		/* eslint-disable-next-line no-nested-ternary */
```
Comment (nomego) [inline cosmoz-tabbable-behavior.js]: Drop the disable and write the statement more expressively.

**Comment by cristinecula** [comment]
@Wurper @nomego ready to release!

**Pull request merged**